### PR TITLE
First version of mobile docs

### PR DESCRIPTION
**Pull request by gr0**
Guys, if you could have a look at the text of the mobile docs it would be awesome. We will be changing screenshots here, but I would like to discuss the text and if it is correct add the same or very similar one to README for both iOS and Android libraries. 

The docs page itself looks like this:
mobile docs

### CODE REVIEW

File: docs/agents/mobile.md [hunk 1]
```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
...
+- automatic and manual geolocation tagging
 - ship logs in configurable bulk requests to minimize battery and network usage
 - buffer logs while the device is disconnected from the network to avoid data loss
 - additional parameters to control when logs should or should not be shipped
-- meta data with application name and version, OS name and version, and application UUID
+- pause and resume log shipping on demand
+- metadata with application name and version, OS name and version, and application UUID
 - ability to catch and ship uncaught exceptions
 
+Use Sematext Cloud **Mobile Application Logs** Logs App type to get out of the box reports giving you the most important information regarding your mobile applications.
```
Comment (otisg) [inline docs/agents/mobile.md:15]: Logs App type => App
giving you => with
regarding => about

4th grade level words :)

File: docs/agents/mobile.md [hunk 2]
```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
...
+ - error log events
+
+<img
+  class="content-modal-image"
+  alt="Mobile Logs Errors"
+  src="../../images/agents/mobile_errors.png"
+  title="Mobile Logs Errors"
+/> 
+
+Get an insight by looking into the dedicated Android and iOS reports that include:
```
Comment (otisg) [inline docs/agents/mobile.md:42]: Get an insight by looking into the dedicated Android and iOS reports that include => Get insights from dedicated Android and iOS reports that include

File: docs/agents/mobile.md [hunk 3]
```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
...
+
+ - mobile operating system logs histogram and their count
```
Comment (otisg) [inline docs/agents/mobile.md:44]: Is this a latency histogram or ...?

File: docs/agents/mobile.md [hunk 4]
```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
...
+ - top severities, version, version codes, and version names
```
Comment (otisg) [inline docs/agents/mobile.md:45]: top severities, version, => top severities, versions,

File: docs/agents/mobile.md [hunk 5]
```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
...
+ - mobile operating system log events
```
Comment (otisg) [inline docs/agents/mobile.md:46]: Hm, mobile operating system sounds like the OS logs, and not just those from the mobile app, are being collected...

File: docs/agents/mobile.md (hunk 1 above)
Comment (gr0) [reply docs/agents/mobile.md:15]: Done, thanks :)

File: docs/agents/mobile.md (hunk 2 above)
Comment (gr0) [reply docs/agents/mobile.md]: Done, thanks!

File: docs/agents/mobile.md (hunk 3 above)
Comment (gr0) [reply docs/agents/mobile.md:44]: No, this is not latency, it is count histogram. Added.

File: docs/agents/mobile.md (hunk 4 above)
Comment (gr0) [reply docs/agents/mobile.md:45]: Done.

File: docs/agents/mobile.md (hunk 5 above)
Comment (gr0) [reply docs/agents/mobile.md:46]: Good point, changed to `mobile applications log events`.

File: docs/agents/mobile.md [hunk 6]
```diff
@@ -4,13 +4,55 @@ description: Ship logs from iOS and Android apps to Sematext
...
+- automatic and manual geolocation tagging
 - ship logs in configurable bulk requests to minimize battery and network usage
 - buffer logs while the device is disconnected from the network to avoid data loss
 - additional parameters to control when logs should or should not be shipped
-- meta data with application name and version, OS name and version, and application UUID
+- pause and resume log shipping on demand
+- metadata with application name and version, OS name and version, and application UUID
 - ability to catch and ship uncaught exceptions
 
+Use Sematext Cloud **Mobile Application Logs** App to get out of the box reports with the most important information about your mobile applications.
```
[the remark this answers is not in the document]
Comment (gr0) [reply docs/agents/mobile.md]: Thanks, adjusted!

**Comment by gr0** [comment]
The screenshots were now updated and it is waiting for the Logpack GEO to be ready. Once we merge it we can merge the docs as well.

**Comment by gr0** [comment]
Btw, the iOS and Android libraries README files were also updated and are waiting to be merged.

**Pull request merged**